config/sync/core.extension.yml

```diff
@@ -11,6 +11,7 @@ module:
   config_translation: 0
   content_translation: 0
   contextual: 0
+  csv_serialization: 0
```

Comment:
Just for my understanding, what do the `0`s mean in this config?

Reply:
`0` means enabled!  Makes sense right?

It's actually the weight, so you can have modules loaded in last.  But most will be 0, which makes things pretty confusing.

Reply:
Essentially this file controls what modules/profiles/themes are enabled.  So to enable a new module, it just needs a line added to this file.